package.json

```diff
@@ -106,10 +106,11 @@
     "mini-css-extract-plugin": "0.11.3",
     "minimist": "1.2.5",
     "optimize-css-assets-webpack-plugin": "5.0.3",
-    "postcss-flexbugs-fixes": "4.2.1",
+    "postcss": "8.3.5",
+    "postcss-flexbugs-fixes": "5.0.2",
     "postcss-global-import": "1.0.6",
-    "postcss-loader": "3.0.0",
-    "postcss-normalize": "9.0.0",
+    "postcss-loader": "4.2.0",
+    "postcss-normalize": "10.0.1",
     "postcss-preset-env": "6.7.0",
     "postcss-resolution-independence": "1.0.0",
     "postcss-safe-parser": "5.0.2",
```

Comment:
Please remove "sanitize.css" from the list.

Reply:
ok